src/main/java/org/kordamp/json/xml/XMLSerializer.java

```diff
@@ -974,6 +980,12 @@ private boolean isNullObject(Element element) {
                 && (element.getAttribute(addJsonPrefix("class")) != null && element.getAttribute(addJsonPrefix("type")) != null)) {
                 return true;
             }
+//            String clazz = getClass(element);
```

Comment:
These are new code, which is introduced as part of development, should be cleaned up from code base.